airbyte-integrations/connectors/source-mailgun/acceptance-test-config.yml

```
@@ -1,24 +1,37 @@
-# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
-# for more information about how to configure these tests
 connector_image: airbyte/source-mailgun:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_mailgun/spec.json"
+    tests:
+      - spec_path: "source_mailgun/spec.yaml"
```

This is going to bypass the backward test.
```suggestion
      - spec_path: "source_mailgun/spec.yaml"
        backward_compatibility_tests_config:
          disable_for_version: 0.1.1